### PR TITLE
Add in Geometry.getintersection api method to find what geometries in…

### PR DESCRIPTION
**Pull request by seamuslee001**
…tersect

### CODE REVIEW

File: api/v3/Geometry.php [hunk 1]
```diff
@@ -437,6 +437,42 @@ function _civicrm_api3_geometry_unarchive_spec(&$spec) {
...
+ * This is used for documentation and validation.
+ *
+ * @param array $spec description of fields supported by this API call
+ * @return void
+ * @see http://wiki.civicrm.org/confluence/display/CRMDOC/API+Architecture+Standards
+ */
+function _civicrm_api3_geometry_getintersection_spec(&$spec) {
+  $spec['geometry_a'] = [
+    'title' => E::ts('Geometry A'),
+    'api.required' => 1,
```
Comment (johntwyman) [inline api/v3/Geometry.php]: @seamuslee001 do we need this `api.required` value for geometry B as well?

Comment (seamuslee001) [reply api/v3/Geometry.php]: @johntwyman i would argue no lets say you want to find all the State electorates that ward is in, you can specify the ward as geometry_a and then the ward collection as collection-id with no specific geometry_b

Comment (johntwyman) [reply api/v3/Geometry.php]: Ah ok. That's a use case that hadn't occurred to me. As long as you pass either a geometry or a collection, then you can proceed. Makes sense. 👍

Comment (seamuslee001) [reply api/v3/Geometry.php]: @johntwyman that has now been implemented in new Line 448 of this file

**Pull request merged**